### DISCUSSION TITLE
Installation error: rm2 (2.15.1.1189)

### DISCUSSION THREAD

**Discussion by PepikVaio** [toltec-dev/toltec#783] [Support] [answered]
Hi, I am trying install toltec and I am getting this error:

root@reMarkable:~# wget http://toltec-dev.org/bootstrap
Connecting to toltec-dev.org (5.161.99.157:80)
saving to 'bootstrap'
bootstrap            100% |****************************************************************************************|  8491  0:00:00 ETA
'bootstrap' saved
root@reMarkable:~# echo "04a28483286f88c5c7f39e352afb62adc57f6162a29fd7e124d832205bb0980e bootstrap" | sha256sum -c && bash bootstrap
sha256sum: WARNING: 1 of 1 computed checksums did NOT match
root@reMarkable:~#

**Comment by PepikVaio** [comment]
Sorry - I was connected via cable.

**Comment by Eeems** [comment, marked as answer]
`rm bootstrap` and then try again. Something has corrupted the file during transmission.
